team.md

```diff
@@ -0,0 +1,239 @@
+#### Create table team(tid, country, totalmatch, win,loss). Insert 8 team detail. Create trigger to verify totalmatch is not 0. export data into csv file. Read file using reader object and print record with winning %age.
+
+
+```python
+import sqlite3 as sq
+```
+
+
+```python
+con=sq.connect("D:/23BCA296/sqlite-tools-win-x64-3460000/team.db")
+```
+
+
+```python
+cur=con.cursor()
+```
+
+
+```python
+q="create table team(tid, country, totalmatch, win,loss)"
+```
+
+
+```python
+cur.execute(q)
+```
+
+
+
+
+    <sqlite3.Cursor at 0x2aef0d422c0>
+
+
+
+
+```python
+con.commit
+```
+
+
+
+
+    <function Connection.commit()>
+
+
+
+
+```python
+con.close
+```
+
+
+
+
+    <function Connection.close()>
+
+
+
+### insert record
+
+
+```python
+cur=con.cursor()
+```
+
+
+```python
+cur.execute("insert into team values(1, 'USA', 10, 7, 3)")
+```
+
+
+
+
+    <sqlite3.Cursor at 0x2aef0d41340>
+
+
+
+
+```python
+cur.execute("insert into team values(2, 'Canada', 12, 8, 4)")
```

Comment:
In integer(' ') why not use it?

Reply:
Even if you could, it wouldn't make sense to use INTEGER(' ') because ' ' is a string value, not an integer value.